src/app/components/core/notifications/notification-service-modal/notification-service-modal.component.html

```diff
@@ -0,0 +1,52 @@
+<div id="notification-setting-modal" class="modal">
```

Comment:
Rename  `notification-setting-modal` -> `notification-service-modal` for naming consistency.